### DISCUSSION TITLE
Does it support postgresql index with some feature?

### DISCUSSION THREAD

**Discussion by athxx** [prisma/prisma#19555] [Q&A] [answered]
I have tables with some columns base on `point` type, and need to create `gist()` index, and turn on buffering. 

the SQL is : 
```sql
create index IdxGeoUser on GeoStore using gist(pt) with (buffering=on);
```
but there seems no way to create `with (buffering=on)` this attribute on `schema.prisma` file
```prisma
model GeoUser {
    uid BigInt               @id @default(autoincrement())
    pt  Unsupported("point")

    @@index([pt], type: Gist) // ??? how to add `with (buffering=on)`
}
```

Is there any way can solve this problem?

**Comment by ludralph** [comment, marked as answer]
Hi @athxx 👋 

To add the `with (buffering=on)` you will need to edit the migration file and add it there.

You can do so by using the command `npx prisma migrate dev --create-only` and press enter then give the migration a name and you should see this

```
Prisma Migrate created the following migration without applying it 20230601121908_init

You can now edit it and apply it by running prisma migrate dev.
```
From here, you can go and edit the migration file like 

```sql
CREATE INDEX "GeoUser_pt_idx" ON "GeoUser" USING GIST ("pt") WITH (buffering=on);
```
Once you're done editing the migration file, you can run the command  ` npx prisma migrate dev` and this should apply the migration.

**Reply by nurul3101** [reply]
@athxx Did you had a chance to look at @ludralph's response?

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
